avocado/core/nrunner.py

```diff
@@ -31,6 +34,16 @@ def __init__(self, kind, uri, *args, **kwargs):
         self.args = args
         self.tags = kwargs.pop('tags', None)
         self.kwargs = kwargs
+        #: Wether this runnable needs a managed temporary directory, which
+        #: will be created and cleaned up by the runner.  The goal of such
+        #: a temporary directory is to hold content that the test will create
+        #: durings its workings, but that can be removed later.  This is
+        #: similar in purpose to :meth:`avocado.Test.workdir`, and can be
+        #: used on tasks such as decompressing source tarballs, building
+        #: software, etc.  If it's set to True, it means that a work_dir is
+        #: required.  It can also be set to a specific location to be used
+        #: as a work directory, if the runnable already has one crated.
+        self.work_dir = False
```

Comment:
maybe this could be None, instead of False.

Reply:
The idea of setting it to `False` is to be a bit more explicit about it not being a requirement.  But, I'll think a bit more about it and surely consider your comment.